class.jetpack-gutenberg.php

```diff
@@ -801,4 +815,46 @@ public static function get_extensions_preset_for_variation( $preset_extensions_m
 
 		return $preset_extensions;
 	}
+
+
+	/**
+	 * Set the availability of the block as the editor
+	 * is loaded.
+	 *
+	 * @param string $slug Slug of the block.
+	 */
+	public static function set_availability_for_plan( $slug ) {
+		$is_available = true;
+		$plan         = '';
+
+		if ( defined( 'IS_WPCOM' ) && IS_WPCOM ) {
+			if ( ! class_exists( 'Store_Product_List' ) ) {
+				require WP_CONTENT_DIR . '/admin-plugins/wpcom-billing/store-product-list.php';
+			}
+			$features_data = Store_Product_List::get_site_specific_features_data();
```

Comment:
Should we pass in the site_id, or if this isn't site specific then can we just use `get_feature_list`

Reply:
We can, but the default is to call `get_current_blog_id()` if the ID isn't passed in. It is the site-specific data we need as it gives a collection of active features and available ones, including the plans that the site would need to upgrade to.